packages/wrangler/src/dev/create-worker-preview.ts

```diff
@@ -172,7 +176,8 @@ export async function createPreviewSession(
 		exchangeResponse.status
 	);
 	logger.debug("HEADERS:", JSON.stringify(exchangeResponse.headers, null, 2));
```

Comment:
I like this line of thinking but... it would necessitate a sanitisation heuristic

I think the public API being the `WRANGLER_LOG_SANITIZE` env var and the contract being sensitive info is not written to the log file – those are covered by this PR and we can think about a richer implementation (+ heuristic) later with more time

Reply:
I think long-term we can think about some sort of heuristic along the lines of high-entropy text gets redacted by default